src/node/interfaces.cpp

```diff
@@ -281,6 +281,13 @@ class NodeImpl : public Node
         }
         return false;
     }
+    std::map<CNetAddr, LocalServiceInfo> getLocalAddresses() override
+    {
+        std::map<CNetAddr, LocalServiceInfo> empty;
+
+        return m_context->connman ? m_context->connman->getLocalAddresses() : empty;
```

Comment:
Thanks for all the support and patient. Applying them. The `{}` was also my first try but I got:
`error: initializer list cannot be used on the right hand side of operator ':'`
Now I changed it to 
```
        if (m_context->connman)
            return m_context->connman->getLocalAddresses();
        else
            return {};
```
to prevent that ugly `empty` variable of mine.